src/AMDGPU.jl

```diff
@@ -17,7 +17,7 @@ import Core: LLVMPtr
 ### Exports ###
 
 export HSAAgent, HSAQueue, HSAExecutable, HSAKernelInstance, HSASignal
-export get_agents, profile, get_first_isa_string, launch!
+export get_agents, profile, get_first_isa_string, launch
```

Comment:
```suggestion
export get_agents, profile, get_first_isa_string
```

Let's just get rid of this so we don't conflict at all.